libs.ts

```diff
@@ -1543,18 +1543,15 @@ export const libs: Lib[] = [
 		permsv2: 'Yes',
 		automod: {
 			text: 'Has a PR',
-			url: 'https://github.com/hikari-py/hikari/pull/1220'
+			url: 'https://github.com/hikari-py/hikari/pull/2219'
```

Comment:
looks like this should be
```suggestion
			url: 'https://github.com/hikari-py/hikari/pull/2205'
```
? https://github.com/hikari-py/hikari/pull/2205

Reply:
Oh yeah, I might have copied the wrong number :P